egov-indexer/notification-order-indexer.yml

```
@@ -0,0 +1,43 @@
+              - inJsonPath: $.date
+                outJsonPath: $.Data.orderNotification.date
+              - inJsonPath: $.entityType
+                outJsonPath: $.Data.orderNotification.entityType
+              - inJsonPath: $.title
+                outJsonPath: $.Data.orderNotification.title
+              - inJsonPath: $.filingNumbers
+                outJsonPath: $.Data.orderNotification.filingNumbers
+              - inJsonPath: $.caseNumbers
+                outJsonPath: $.Data.orderNotification.caseNumbers
+              - inJsonPath: $.judgeIds
+                outJsonPath: $.Data.orderNotification.judgeIds
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)